packages/cosmos_ui_components/lib/cosmos_theme.dart

```diff
@@ -87,6 +88,68 @@ class CosmosThemeData extends Equatable {
     this.errorColor = CosmosColorsData.defaultError,
   });
 
+  const CosmosThemeData.dark({
+    this.spacingXXXL = defaultSpacingXXXL,
+    this.spacingXXL = defaultSpacingXXL,
+    this.spacingXL = defaultSpacingXL,
+    this.spacingL = defaultSpacingL,
+    this.spacingM = defaultSpacingM,
+    this.spacingS = defaultSpacingS,
+    this.spacingXS = defaultSpacingXS,
+    this.longDuration = defaultLongDuration,
+    this.mediumDuration = defaultMediumDuration,
+    this.shortDuration = defaultShortDuration,
+    this.radiusXL = defaultRadiusXL,
+    this.radiusL = defaultRadiusL,
+    this.radiusM = defaultRadiusM,
+    this.radiusS = defaultRadiusS,
+    this.fontSizeS = defaultFontSizeS,
+    this.fontSizeM = defaultFontSizeM,
+    this.fontSizeL = defaultFontSizeL,
+    this.fontSizeXL = defaultFontSizeXL,
+    this.fontSizeXXL = defaultFontSizeXXL,
+    this.borderRadiusL = defaultBorderRadiusL,
+    this.borderRadiusM = defaultBorderRadiusM,
+    this.borderRadiusS = defaultBorderRadiusS,
+    this.elevationS = defaultElevationS,
+    this.elevationM = defaultElevationM,
+    this.elevationL = defaultElevationL,
+    this.colors = darkThemeColors,
+    this.brightness = Brightness.dark,
+    this.errorColor = CosmosColorsData.defaultError,
+  });
+
+  const CosmosThemeData.light({
+    this.spacingXXXL = defaultSpacingXXXL,
+    this.spacingXXL = defaultSpacingXXL,
+    this.spacingXL = defaultSpacingXL,
+    this.spacingL = defaultSpacingL,
+    this.spacingM = defaultSpacingM,
+    this.spacingS = defaultSpacingS,
+    this.spacingXS = defaultSpacingXS,
+    this.longDuration = defaultLongDuration,
+    this.mediumDuration = defaultMediumDuration,
+    this.shortDuration = defaultShortDuration,
+    this.radiusXL = defaultRadiusXL,
+    this.radiusL = defaultRadiusL,
+    this.radiusM = defaultRadiusM,
+    this.radiusS = defaultRadiusS,
+    this.fontSizeS = defaultFontSizeS,
+    this.fontSizeM = defaultFontSizeM,
+    this.fontSizeL = defaultFontSizeL,
+    this.fontSizeXL = defaultFontSizeXL,
+    this.fontSizeXXL = defaultFontSizeXXL,
+    this.borderRadiusL = defaultBorderRadiusL,
+    this.borderRadiusM = defaultBorderRadiusM,
+    this.borderRadiusS = defaultBorderRadiusS,
+    this.elevationS = defaultElevationS,
+    this.elevationM = defaultElevationM,
+    this.elevationL = defaultElevationL,
+    this.colors = const CosmosColorsData(),
+    this.brightness = Brightness.light,
+    this.errorColor = CosmosColorsData.defaultError,
```

Comment:
btw. this errorColor should be part of `CosmosColorData`

Reply:
This was a part of `CosmosColorsData` but we it had the same value for both themes, so we decided it to keep it a part of `CosmosTheme`

Reply:
hm, not sure I get it, even if its the same for both themes, it still can reside in `CosmosColorData`, as this is the place for colors basically